molgenis-js/src/main/resources/js/script-evaluator.js

```diff
@@ -20,11 +19,24 @@ function evalScript (script, entity) {
        * Example: $('Height').value() returns the values of the height
        * attribute
        *
+       * If the value is an object, return the idValue
+       *
        * @memberof $
        * @method value
        *
        */
       value: function () {
+        if (this.val !== null && typeof this.val === 'object') {
```

Comment:
really need a unit test of this ? or is there one already ? code cov does not pick find this one i would guess